pages/What is indentation and why does it matter%3F.md

```diff
@@ -1,13 +1,13 @@
-title:: What is indentation and why does it matter?
+title:: Using Indentation
 
-- Indentation is one of the fundamental concept in Logseq (and outliners in general). It's the feature that allows block references and queries to work. In other words: indentation helps you to find stuff in Logseq.
-- The idea of indentation is simple: anytime you hit the `Tab` key, the block your cursor is in moves to the right. Indenting a block _nests_ it under the block above, creating a parent-child relationship:
+- Indentation is an important visual representation in Logseq (and outliners in general). It allows an view of a hierarchical section of your graph on a page. Representing hierarchies (mind maps, file directories, taxonomies, course prerequisites, org charts, etc) is an important visual tool. Logseq's indentation allows for this type of on-page presentation.
```

Comment:
The original intro is focused on basic usage as this is meant to be beginner friendly. Replacing it with mentions of mind maps and hierarchies is going to confuse them and take the focus away from the tutorial's  intent